tests/no_normal_tasks.rs

```diff
@@ -26,14 +26,16 @@ fn test_no_normal_tasks() {
     let mock_login = mock(POST, LOGIN_PATH).return_status(200).create();
     let mock_logout = mock(GET, LOGOUT_PATH).return_status(200).create();
 
-    crate::GooseAttack::initialize_with_config(common::build_configuration())
+    let _stats = crate::GooseAttack::initialize_with_config(common::build_configuration())
```

Comment:
Technically it's a goose attack and not stats ...

Reply:
fixed